scheduler/src/cook/kubernetes/api.clj

```diff
@@ -460,21 +466,40 @@
                            {})]
     (assoc resource-map :gpus gpu-model->count)))
 
+(defn pool->disk-type-label-name
+  "Given a pool name, get the disk-type-label-name that will be used as a nodeSelector label"
+  [pool-name]
+  (regexp-tools/match-based-on-pool-name
+    (config/disk)
+    pool-name
+    :disk-node-label
+    :default-value "cloud.google.com/gke-boot-disk"))
+
+(defn add-disk-type-to-resource-map
+  "Given a map from node-name->resource-type->capacity, set the disk capacity to type->amount"
```

Comment:
I'd rename the function to force-disk-type-in-resource-map

And then change the docstring to:
"Given a map from node-name->resource-type->capacity, set the disk capacity to type->amount or remove the disk resource type from the map if disk-type is not provided"

Reply:
agree on docstring, but similar function for gpus is `add-gpu-model-to-resource-map`, would you prefer changing that function name to `force-gpu-model-in-resource-map` too?

Reply:
interesting. add-gpu-model-to-resource-map doesn't delete the :gpus key, just points it to an empty map

I think they should both be called "force" and both delete the key. I like not having the key better than pointing to an empty map

Reply:
done